openstef_dbc/data_interface.py

```diff
@@ -35,17 +36,20 @@ def __init__(self, config):
                 influxdb_host (str): InfluxDB host.
                 influxdb_port (int): InfluxDB port.
                 influx_organization (str): InfluxDB organization.
-                mysql_username (str): MySQL username.
-                mysql_password (str): MySQL password.
-                mysql_host (str): MySQL host.
-                mysql_port (int): MYSQL port.
-                mysql_database_name (str): MySQL database name.
+                sql_db_username (str): SQL database username.
+                sql_db_password (str): SQL database password.
+                sql_db_host (str): SQL database host.
+                sql_db_port (int): SQL database port.
+                sql_db_database_name (str): SQL database name.
                 proxies Union[dict[str, str], None]: Proxies.
         """
 
         self.logger = logging.get_logger(self.__class__.__name__)
         self.influx_organization = config.influx_organization
 
+        # Get db type from config, set 'mysql' if the variable does not exist
+        self.sql_db_type = getattr(config, 'sql_db_type', 'mysql').lower()
```

Comment:
Why do we need to call lower() here? If we can only handle two options maybe better to define this as an enum.

Reply:
I removed it and the other places where we put upper for log purposes it seems useless